From 79230bbb3ad191628da205a4c5bc153c802e1da6 Mon Sep 17 00:00:00 2001
From: Richard Zhao <richard.zhao@freescale.com>
Date: Wed, 9 Nov 2011 15:01:45 +0800
Subject: ENGR00161764 mx6q: disable gc355 temporally

Reasons:
 - gc355 suspend failed. which cause the whole system suspend failed.
 - gc355 cause reserved memory must be ioremap, consume too much vmalloc space.
 - gc355 can not work in 4.6.2 gpu driver.

Signed-off-by: Richard Zhao <richard.zhao@freescale.com>
---
 arch/arm/plat-mxc/devices/platform-viv_gpu.c |    2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/arch/arm/plat-mxc/devices/platform-viv_gpu.c b/arch/arm/plat-mxc/devices/platform-viv_gpu.c
index b11efe0..7108cac 100644
--- a/arch/arm/plat-mxc/devices/platform-viv_gpu.c
+++ b/arch/arm/plat-mxc/devices/platform-viv_gpu.c
@@ -28,7 +28,7 @@ const struct imx_viv_gpu_data imx6_gpu_data __initconst = {
 	.iobase_2d = GPU_2D_ARB_BASE_ADDR,
 	.irq_2d = MXC_INT_GPU2D_IRQ,
 	.iobase_vg = OPENVG_ARB_BASE_ADDR,
-	.irq_vg = MXC_INT_OPENVG_XAQ2,
+	.irq_vg = -1 /*MXC_INT_OPENVG_XAQ2*/,
 };
 #endif
 
-- 
1.7.9.5

